lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -1105,6 +1107,30 @@ def configure_hardware_params(self, vm_obj):
                         # Don't fail if VM is already upgraded.
                         pass
 
+            if 'virt_based_security' in self.params['hardware']:
+                guest_ids = ['windows9_64Guest', 'windows9Server64Guest']
```

Comment:
Can we also add additional check if ESXi is 6.7 and greater so that we can set it ? For older ESXi version this will fail.

Reply:
Added the ESXi version check. 
Does this test env support specify hardware version when deploy vm? If not, should I comment out this test file in main.yml?